WickedEngine/wiRenderer.cpp

```diff
@@ -54,7 +54,12 @@ GPUBuffer			constantBuffers[CBTYPE_COUNT];
 GPUBuffer			resourceBuffers[RBTYPE_COUNT];
 Sampler				samplers[SSLOT_COUNT];
 
+// Specify shader folder using macro when running from CMake
+#ifdef WICKED_DEFAULT_SHADERS_DIRECTORY
+string SHADERPATH = WICKED_DEFAULT_SHADERS_DIRECTORY;
```

Comment:
This is not good, this would lead to a lot of confusion. You shouldn't modify this. 
Apart from that, it doesn't fix the issue anyway. If it would fix the shaderpath issue, it would still not fix the working directory issue.